src/packages/cli/src/CLI.ts

```diff
@@ -119,9 +117,13 @@ export class CLI implements Command {
                 init   Setup Prisma for your app
           introspect   Get the datamodel of your database
             generate   Generate artifacts (e.g. Prisma Client)
-              format   Formats your schema
+              studio   Open Prisma Studio
```

Comment:
Why this unrelated change?

Reply:
You mean adding Studio? We forgot to do that when we switched Studio to GA 🤷‍♂️